src/Relogify.Android/MainActivity.fs

```diff
@@ -12,14 +12,63 @@ open Android.Widget
 open Android.OS
 open Xamarin.Forms.Platform.Android
 
+open Android.Util
+open Android.Gms.Common
+
 [<Activity (Label = "Relogify.Android", Icon = "@mipmap/icon", Theme = "@style/MainTheme", MainLauncher = true, ConfigurationChanges = (ConfigChanges.ScreenSize ||| ConfigChanges.Orientation), ScreenOrientation = ScreenOrientation.Portrait)>]
 type MainActivity() =
     inherit FormsAppCompatActivity()
+
+    let TAG: string = "MainActivity"
+    static member CHANNEL_ID: string = "my_notification_channel" // TODO: "Challenges from other players"
+    static member NOTIFICATION_ID: int = 123
+
+    member private this.CreateNotificationChannel () =
+        if Build.VERSION.SdkInt >= BuildVersionCodes.O then
+            let channelName = MainActivity.CHANNEL_ID
+            let channelDescription = ""

```

Comment:
Set channel description (e.g. "Challenges from other players in the community").